src/persistence/index.js

```diff
@@ -239,6 +245,15 @@ export class DocumentDB {
     }
 }
 
+export class RestoreError extends Error {
+    constructor(cause, value) {
+        super(`Cannot restore: ${cause.message}`)
+
```

Comment:
// Set the prototype explicitly
Object.setPrototypeOf(this, PersistanceError.prototype)